src/app/routes/auth/containers/Login.js

```diff
@@ -93,16 +103,27 @@ class Login extends React.Component {
                 /*
                     All validation has been successful
                     Attempt to log in user
-
-                    - data:
-                        email: this.refs.email.value
-                        password: this.refs.password.value
                 */
-                this.props.attemptLogin(this.refs.email.value, this.refs.password.value)
+                this.props.attemptLogin(this.state.auth.email, this.state.auth.password)
             }.bind(this)
         };
     }
 
+    onChange(event) {
+        /*
+            Event method for onChange input event
+        */
+        event.preventDefault();
+        var newState = {};
+        if (event.target.name === 'password') {
```

Comment:
should we crate two different functions for change events for the password and email?